app/dashboard/template.hbs

```diff
@@ -27,7 +27,7 @@
     </div>
     <div class="quickSearch">
         <div class="container p-t-lg">
-            <div class="row m-t-lg">
+            <div class="row m-t-lg {{if (gt  nodes.length 9) 'quick-search-contents'}}">
```

Comment:
I'm not sure the best approach here, but this still results in some jumping, especially when filtering:
![kapture 2018-02-02 at 13 32 17](https://user-images.githubusercontent.com/6776190/35749002-0b21576c-081e-11e8-997d-647e210a1221.gif)

Someone stop me if I'm overdoing this, but it might help to change around how/when things are displayed.

Currently (both in legacy OSF and this PR), the search box and table header are hidden until the projects load. With our improved async handling, we could display them from the start and allow the user to enter stuff in search before we have the first page of projects, if they want. I think seeing this (maybe without the next page button) right away would be more friendly:
![kapture 2018-02-02 at 13 46 52](https://user-images.githubusercontent.com/6776190/35749453-8e5516fe-081f-11e8-903c-14821f3a2aa7.gif)
and it'd be nicely consistent to use that same view when loading a new sort/filter. (It might ideally be a skeleton screen, but it's reasonable to put that beyond the scope of this ticket...)

Reply:
I've added a check to prevent jumping up while loading results. I disagree that we want to be showing the search bar and the empty table prior to initial load. Initial load makes sure the user has projects. In your solution, any user with less than 10 project will experience a "jump up" and any user without projects would experience a big jump up, then the search bar would disappear. So, if we can agree that a little bit of jumping cannot be prevented (either that or just have a whole lot of blank space at all times), it should be a priority to minimize jumping (by adding the min-height), and minimize "jump-back" (when the area shrinks only to be extended again, by adding the loading checker). Also, jumping "down" would seem to me preferable to jumping "up", one represents loading of more information, the other that there are less items than "expected". A different solution would be to change the UI significantly as well, as you suggested with the skeleton screen. Not against those changes, although I am very much against those changes resulting from code review conversations, rather than a proper product-approved iteration. Been to enough product demos to understand that changes do happen midway through the process, but it is much easier to work when the change comes from product to developer, rather than the other way around, as well intended as it might be. Much like what is being done to the contributor page, it can be argued that the dashboard requires revision (it does), but if that is the case, I would rather hold work on this until those conversations are had.

Reply:
The language changes make me wary as they are, although they fall under "fixing prior mistakes", as it has been pointed out they are wrong according to our own guides, and consistency is a good thing.

Reply:
Yeah that makes sense, I hadn't thought about users with no projects. Minimizing jump-back and otherwise leaving the UI as-is works for me 👍  

Hopefully we can get a good redesign at some point in the future, but you're right, this is neither the time nor the place.

Reply:
Specially the addition of skeleton screens. I am personally a big fan of them, and if we could work on implementing them uniformly that would be great.